epictrack-web/src/components/indigenousNation/IndigneousNationForm.tsx

```diff
@@ -163,10 +163,7 @@ export default function IndigenousNationForm({ ...props }) {
             <TextField fullWidth {...register("pip_link")} />
           </Grid>
           <Grid item xs={6} sx={{ paddingTop: "30px !important" }}>
-            <ControlledSwitch
-              defaultChecked={(ctx.item as FirstNation)?.is_active}
-              {...register("is_active")}
-            />
+            <ControlledSwitch {...register("is_active")} />
```

Comment:
I think the ...register(..) should be added in the ControlledSwitch itself. like:
![image](https://github.com/bcgov/EPIC.track/assets/91914654/cb62f431-4cda-4d86-a996-e18d1085c226)